### DISCUSSION TITLE
About using prometheus to monitor miluvs and display performance indicators

### DISCUSSION THREAD

**Discussion by faceall666** [milvus-io/milvus#5135] [Q&A and General discussion]
集群需要采用prometheus + grafana 的监控方案，但是调试中为接收到milvus 文档中指定的性能指标等参数，请问是否有确切的对应关系文档

下图中的指标在prometheus中的字段是什么。
![image](https://user-images.githubusercontent.com/83273334/117429380-957ed700-af59-11eb-96ae-6e82fb5fa33b.png)

**Comment by fishpenguin** [comment]
These fields are corresponding to the title of each metric in grafana dashboard. Currently, there isn't any corresponding docs between prometheus and these metrics, because we use promQL to get some statistic metrics from prometheus data, in other words, these metrics correspond to the result of a promQL but not simple prometheus data.
